examples/next/pages/ui/hooks/useDataStore/sort/Collection.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { User } from '../models';
+import {
+  createDataStorePredicate,
+  getOverrideProps,
+  useDataStoreBinding,
+} from '@aws-amplify/ui-react/internal';
+import { SortDirection } from '@aws-amplify/datastore';
+import { Collection, Flex, Text } from '@aws-amplify/ui-react';
+import { userFilterPredicate } from '../utils';
+
+const CollectionWithSort = (props) => {
+  const { backgroundColor, items, overrideItems, overrides, ...rest } = props;
```

Comment:
Similar to my comment with `CollectionAndRecord`, think this component should be simplified to the point where it receives no props

Reply:
Good call!